setup.py

```diff
@@ -120,6 +120,7 @@ def find_version(*file_paths):
                     "tqdm>=4.54.1",
                     "natsort>=7.1.0",
                     "pandas>=1.1.5,<1.4.0rc0",
+                    "tabulate>=0.8.3",
```

Comment:
@vuiseng9 Please, make sure there are no license issues. 

cc' @MaximProshin, @ljaljushkin, @vshampor

Reply:
@alexsu52 , good catch! we need to scan it by Snyk and other tools before the merge.